js/src/dapps/registry/Container.js

```diff
@@ -8,23 +8,30 @@ import * as actions from './actions';
 class Container extends Component {
   static propTypes = {
     actions: PropTypes.object,
+    accounts: PropTypes.object,
+    account: PropTypes.object,
     contract: PropTypes.object,
     owner: PropTypes.string,
     fee: PropTypes.object,
-    lookup: PropTypes.object
+    lookup: PropTypes.object,
+    events: PropTypes.array
   };
+
   componentDidMount () {
     this.props.actions.fetchContract();
   }
 
   render () {
-    const { actions, contract, owner, fee, lookup } = this.props;
+    const props = this.props;
     return (<Application
-      actions={ actions }
-      contract={ contract }
-      owner={ owner }
-      fee={ fee }
-      lookup={ lookup }
+      actions={ props.actions }
```

Comment:
https://facebook.github.io/react/docs/transferring-props.html#transferring-with-...-in-jsx